roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -1,5 +1,5 @@
 {
-    "context_is_admin":  "role:admin",
+    "context_is_admin":  "role:admin or role:cloud_admin",
     "admin_or_owner":  "is_admin:True or project_id:%(project_id)s",
```

Comment:
It would, but this is the part in Nova I mentioned where cloud_admin is being deliberately added to the rule. Otherwise the user will need to be added to projects in order to perform a majority of the actions.